.circleci/config.yml

```diff
@@ -0,0 +1,68 @@
+# https://circleci.com/docs/2.0/configuration-reference/
+defaults: &defaults
+  docker:
+    - image: circleci/python:3.6.5-stretch-node-browsers
+
+version: 2
+jobs:
+# Build Arnold's docker image
+  build:
+    <<: *defaults
+    steps:
+      - checkout
+      # Activate dind
+      - setup_remote_docker:
+          docker_layer_caching: true
+      - run:
+          name: Build arnold production image
+          command: |
+            IMAGE="arnold:$(tr -d '\n' < VERSION)"
+            docker build -t "${IMAGE}" .
+            mkdir -p /tmp/docker/images
+            docker save -o /tmp/docker/images/arnold.tar "${IMAGE}"
+      # Store the built as an artifact that can be reused in other jobs
+      - persist_to_workspace:
+          # Must be an absolute path, or relative path from working_directory.
+          # This is a directory on the container which is
+          # taken to be the root directory of the workspace.
+          root: /tmp/docker
+          # Must be relative path from root
+          paths:
+            - images
+
+  test:
+    <<: *defaults
+    steps:
+      - checkout
+      # Activate dind
+      - setup_remote_docker:
+          docker_layer_caching: true
+      # Restore workspace
+      - attach_workspace:
+          # Must be absolute path or relative path from working_directory
+          at: /tmp/docker
+      - run:
+          name: Test builded image
+          command: |
+            IMAGE="arnold:$(tr -d '\n' < VERSION)"
+            docker load < /tmp/docker/images/arnold.tar
+            docker images
+            echo "oc must be installed"
+            docker run --rm \
+              -e OC_LOGIN=false \
+              "${IMAGE}" \
+              oc version
+            echo "ansible must be installed"
+            docker run --rm \
+              -e OC_LOGIN=false \
+              "${IMAGE}" \
+              ansible --version
```

Comment:
I think it would be better to split those two tests in two separated steps from this job:

```yaml
- run:
    name: Test oc client installation
    ...
- run:
    name: Test ansible installation
    ...
```

Reply:
Grr !